packages/web/src/components/OpenSourceDatasets/index.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+import { useDispatch } from "react-redux";
+
+import DatasetTable from "./DatasetTable";
+import DatasetDetails from "../DatasetDetails";
+import { DatasetManifestUrl } from "../../constants";
+import { metadata } from "../../../../core/state";
+
+import styles from "./OpenSourceDatasets.module.css";
+
+/**
+ * Page for displaying public-facing datasets
+ * Currently using placeholder text and data
+ */
+export default function OpenSourceDatasets() {
+    const dispatch = useDispatch();
+    // Begin update action so dataset manifest is ready for table child component
+    dispatch(
+        metadata.actions.updateDatasetManifest("Dataset Manifest", DatasetManifestUrl.PRODUCTION)
```

Comment:
This action seems like a `getter` to me similar to how we do `getAnnotations` - also looks like this is relying on the component `OpenSourceDatasets` to only render once to prevent this action triggering multiple times, wrapping in a `useEffect` or similar might be a nice guard.

Reply:
Added useEffect guard but still haven't significantly changed logic for this one; it's trying to imitate the logic used for getDataSources in the way it interacts with the DatabaseService